src/index.ts

```diff
@@ -786,7 +786,7 @@ const metadata = properties(
     lastModified: string(),
     parent: string(),
     pinned: boolean(),
-    type: enumeration("DocumentType", "CollectionType"),
+    type: enumeration("DocumentType", "CollectionType", "TemplateType"),
```

Comment:
For some reason github won't let me comment on lines outside of this, but can you also add this to the `Metadata` interface above?